### DISCUSSION TITLE
How to transfer the storage path?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9181] [General Q&A]
I have multiple storage paths, one of which is to be deactivated after data transfer. What should I do?

**Comment by rjan90** [comment]
Hey!

In the upcoming Lotus v1.17.1 (currently in release candidate) you can [detach storage paths easily](https://lotus.filecoin.io/storage-providers/operate/custom-storage-layout/#detach-storage-paths), and updating the location of sectors to new storage paths without restarting the Storage Provider: https://lotus.filecoin.io/storage-providers/operate/custom-storage-layout/#updating-locations
